docs/Observability_using_OpenTelemetry.md

```diff
@@ -0,0 +1,897 @@
+- **Usage metrics**: It is important to gather metrics which enable customers to:
+  - Identify and report resource use/Utilization details by each of their organisations/domains/customers.
+  - For ISV like scenarios, have clear and controllable boundaries- distinguish between common portions and customer specific portions. For example in Microsoft Fabric the possible control boundaries are: domains, workspaces and compute capacity.
+  - Present information with varied levels of granularity (At the service level and then service + customer level and so on.).
+
+All these metrics can be combined with execution metrics as described in [the above section](#combining-reference-data-with-telemetry-data-for-alerting-and-additional-metrics) to correlate business events with operational events. For example, increased in number of invalid records due to a service outage, drop in customer ratings with increase in system response time, increased failure rate during quarter end for a financial organization etc.
+
+### Sample OTEL Collector configuration file
+
+Refer to [OTEL Collector configuration](https://opentelemetry.io/docs/collector/configuration/), [available exporters](https://opentelemetry.io/docs/languages/python/exporters/) and [End points and environment configuration](https://opentelemetry.io/docs/specs/otel/protocol/exporter/#configuration-options) for more details on available exporters and advanced configuration.
+
+**Where data is sent**:
+
+Given the sample configuration file contents below:
+
+- For [ADX](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/azuredataexplorerexporter) data will be seen in `oteldb` (See [ADX DDLs](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/azuredataexplorerexporter#database-and-table-definition-scripts) ). Table names will be `OTELTraces` for Traces, `OTELLogs` for logs and `OTELMetrics` for metrics.
+- For [App Insights/Azure monitor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/azuremonitorexporter)
+  - Data will be seen in as follows:
+    - Traces:
+      - "dependency" for Spankind in [CLIENT, PRODUCER, INTERNAL]
+      - "requests" for Spankind in [SERVER, CONSUMER]
+    - Logs: "traces"
+    - Metrics: "customMetrics"
+    - Exception events: "exceptions"
+
+The sample Collector configuration below uses an Environment file(`env.cfg`) placed on the machine where Collector is setup.
+
+```bash
+####################
+# file_name: env.cfg
+#####################
+# Azure App Insights
+APPINSIGHTS CONNECTION STRING="<<App-insgights-connection-string>>"
```

Comment:
Locally tested and double quote (`"`) will be treated as a part of string, so need to remove
```suggestion
APPINSIGHTS CONNECTION STRING=<<App-insgights-connection-string>>
```